community/kernel/src/main/java/org/neo4j/kernel/impl/annotations/AnnotationProcessor.java

```diff
@@ -40,25 +36,11 @@
 import javax.tools.FileObject;
 import javax.tools.StandardLocation;
 
-import static org.neo4j.kernel.impl.util.Charsets.UTF_8;
 import static org.neo4j.io.fs.FileUtils.newFilePrintWriter;
+import static org.neo4j.kernel.impl.util.Charsets.UTF_8;
```

Comment:
What's up with the changes around charset? 
See https://github.com/neo4j/neo4j/commit/b13d1b2980548fe4e5d9a633b3457b354d8d19dd#diff-d232b77c0bec2f58f44f5193f4bf3293
This is the only conflict when forward-merging.